packages/web/src/builders/file-server/file-server.impl.ts

```diff
@@ -79,10 +80,18 @@ function getIgnoredGlobs(root: string) {
 
 function createFileWatcher(root: string, changeHandler: () => void) {
   const ignoredGlobs = getIgnoredGlobs(root);
+  const layout = workspaceLayout();
 
-  const watcher = watch(['./apps/**', './libs/**'], {
-    cwd: root,
-  });
+  const watcher = watch(
+    [
+      joinPathFragments(layout.appsDir, '**'),
+      joinPathFragments(layout.libsDir, '**'),
+    ],
+    {
+      cwd: root,
+      ignoreInitial: true,
```

Comment:
How come this was changed?

I'm not sure but I think `ignoreInitial` is for file creations.. not for skipping the first run?

https://github.com/paulmillr/chokidar#path-filtering